lib/tools/apk-signing.js

```diff
@@ -17,28 +18,6 @@ const APKSIGNER_VERIFY_FAIL = 'DOES NOT VERIFY';
 
 let apkSigningMethods = {};
 
-/**
- * Applies the patch, which workarounds'-Djava.ext.dirs is not supported. Use -classpath instead.'
- * error on Windows by creating a temporary patched copy of the original apksigner script.
- *
- * @param {string} originalPath - The original path to apksigner tool
- * @returns {string} The full path to the patched script or the same path if there is
- *                   no need to patch the original file.
- */
-async function patchApksigner (originalPath) {
-  const originalContent = await fs.readFile(originalPath, 'ascii');
-  const patchedContent = originalContent.replace('-Djava.ext.dirs="%frameworkdir%"',
```

Comment:
ah, since https://android.googlesource.com/platform/tools/apksig/+/1337ba77919fe73d4b236648761d985f01aeb437%5E%21/ , the parameter (`-Djava.ext.dirs=`) is no longer used in the apksigner on Win.